### PR TITLE
Eng 894 certs

### PR DESCRIPTION
**Pull request by JGuzman4**

### CODE REVIEW

File: modules/common/certificates/helm/certificates/templates/certificates.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  name: {{ Values.namespace }}-certificate
+  namespace: istio-system
+spec:
+  acme:
+    config:
+    - domains:
+      - {{ Values.cluster_domain }}
+      - *.{{ Values.cluster_domain }}
+      dns01:
+        provider: route53-liatrio
```
Comment (jphayes) [inline modules/common/certificates/helm/certificates/templates/certificates.yaml]: I'm introducing an output in the istio module which will have this provider name so it's not hard-coded

File: modules/common/certificates/helm/certificates/templates/certificates.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: {{ Values.namespace }}-certificate
+  namespace: istio-system
```
Comment (jphayes) [inline modules/common/certificates/helm/certificates/templates/certificates.yaml]: There's an output from the istio module which has this namespace name

File: modules/common/certificates/helm/certificates/templates/certificates.yaml (hunk 1 above)
Comment (JGuzman4) [reply modules/common/certificates/helm/certificates/templates/certificates.yaml]: Is there a branch youre working on @jphayes ?

File: modules/common/certificates/helm/certificates/templates/certificates.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      - *.{{ Values.cluster_domain }}
+      dns01:
+        provider: route53-liatrio
+    commonName: {{ Values.cluster_domain }}
+  dnsNames:
+  - {{ Values.cluster_domain }}
+  - *.{{ Values.cluster_domain }}
+  issuerRef:
+    kind: Issuer
+    name: letsencrypt-dns
```
Comment (jphayes) [inline modules/common/certificates/helm/certificates/templates/certificates.yaml]: I'm introducing an output in the istio module which will have this issuer name so it's not hard-coded

**Pull request merged**